core/src/scss/utilities/mixins/shadow/_box-shadow.scss

```
@@ -1,15 +1,14 @@
 @charset "UTF-8";
 
-//
-// @box-shadow($depth: 'flat')
-//
-// Box shadows for element containers.
-//
-// $depth   - String: One of flat, shallow, medium, medium-deep, deep
-// $stroke  - border css properties
-//
-// Style guide: Mixins.Shadow.Box Shadow
-//
+///
+/// Box shadows for element containers.
+///
+/// @name box-shadow
+///
+/// @param {string} $depth [flat] - One of flat, shallow, medium, medium-deep, deep
+/// @param {css} $stroke [1px solid #C6C6C6] - border css properties
+///
+/// @group shadow
 @mixin box-shadow($depth: 'flat', $stroke: '1px solid #C6C6C6') {
```

```suggestion
@mixin box-shadow($depth: 'paper', $stroke: '1px solid #C6C6C6') {
```
Seems like the default shouldn't be flat (no shadows at all) since the main function of the mixin is to create a box shadow.